### DISCUSSION TITLE
Score replay is missing

### DISCUSSION THREAD

**Discussion by Cootz** [ppy/osu#22552] [Q&A] [answered]
What the heck?

https://user-images.githubusercontent.com/50776304/217504936-f6c81d48-5d59-4bfa-aa8a-8de06bbbe0b2.mp4

A score replay just despaired. I know that this is a dev build but I'm curious if you can **lose** a replay while canceling an import.
Logs:
[input.log](https://github.com/ppy/osu/files/10684356/input.log)
[network.log](https://github.com/ppy/osu/files/10684357/network.log)
[performance.log](https://github.com/ppy/osu/files/10684358/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10684360/runtime.log)
[database.log](https://github.com/ppy/osu/files/10684361/database.log)
[client_26.zip](https://github.com/ppy/osu/files/10684382/client_26.zip)

**Comment by Walavouchey** [comment, marked as answer]
>I'm curious if you can **lose** a replay while canceling an import.

Have you tried reimporting the replay?
